src/index.html

```diff
@@ -46,12 +50,23 @@ <h3 class="modal__heading">Edit Profile</h3>
         <input class="modal__input" id="profile-description-input" type="text" name="description"
           placeholder="Description" minlength="2" maxlength="200" required />
         <span class="modal__error" id="profile-description-input-error"></span>
-        <button class="modal__submit modal__submit_disabled" type="submit">
+        <button class="modal__submit modal__submit_disabled" id="profile-submit-button" type="submit">
           Save
         </button>
       </form>
     </div>
   </div>
+  <div class="modal" id="avatar-edit-modal">
+    <div class="modal__container">
+      <button class="modal__close" type="button" id="avatar-modal-close"></button>
+      <h3 class="modal__heading">Change profile picture</h3>
+      <form class="modal__form" id="avatar-edit-form" novalidate>
+        <input type="text" class="modal__input" id="avatar-link-input" name="link" placeholder="Image link" type="url" required>
```

Comment:
` type="text" `

* Please, validate the `avatar` input as `url` https://snipboard.io/EhZxjc.jpg 

You should add `type="url"` for the avatar input